### PR TITLE
Natasha changes

### LINKED ISSUE #755: [Discovery] Remove call number attribute from advanced search
Apologies if this has been noted previously, but in the advanced search the Call Number attribute still features call numbers beyond the established A-Z of LC. Additional call numbers in the list include: 
[, >, 9, n

The odd call numbers don't seem to be reflected in the item records (at least for those that I checked). 
For example, limiting to > as a call number lead me to this [results page](https://search.library.ualberta.ca/catalog?all_fields=&author=&commit=Search&f_inclusive%5Blc_1letter_facet%5D%5B%5D=%3E&op=AND&range%5Bpub_date%5D%5Bbegin%5D=&range%5Bpub_date%5D%5Bend%5D=&search_field=advanced&sort=score+desc%2C+pub_date_sort+desc%2C+title_sort+asc&subject=&title=&utf8=%E2%9C%93). The first result is [Quand le stage en enseignement déraille](https://search.library.ualberta.ca/catalog/6189442) which has a call number of LB 2157 A3 Q36 2013. 

Additionally, I would suggest that the call numbers attribute be listed in alphabetical order, rather than in order of # of results, as seems to be the case now. I've marked this as 'enhancement' and for the January/April milestone.

### LINKED ISSUE #1088: TOC not wrapping around Tools box
First time I've noticed this. Confirmed in Chrome and Firefox:
https://www.library.ualberta.ca/catalog/5756085

image

### LINKED ISSUE #1127: seeing Alberta Health Services - Women's Health [in transition] in list view as library name for NLC_INT
in transition
 
The item's true location is Northern Lakes College.

### LINKED ISSUE #1135: [Discovery] MAP_OS location is not mapped / displays as blank
Add MAP_OS location so that displays in the holdings table

e.g. https://www.library.ualberta.ca/catalog/6804133

### LINKED ISSUE #1254: Reword text on Advanced Search page
Remove the following text (first two bullet points) on the [advanced search page](https://library.ualberta.ca/advanced):

- NB: This advanced search screen only searches local content.
- Please use the Articles Advanced Search to search for non-local content.

Suggest replacing the rest verbatim with text from [Stanford Searchworks Advanced Search](https://searchworks.stanford.edu/advanced), but we'll need to verify that all the operators are correct given that they are on a different Blacklight version

### LINKED ISSUE #1268: Fix diacritics font issues
Characters with diacritics have the appearance of being bolded in facet lists, search results, and record views. Some examples from UAT:

**Search Results**
http://uat.library.ualberta.ca:3002/catalog?q=music

**Facet Lists**
capture

**Record Views**
http://uat.library.ualberta.ca:3002/catalog/kule465339893
 - In title, contents notes, etc.

### PR DESCRIPTION
**Pull request by nnunn**
Resolves #1268 
Resolves #1254 
Resolves #1135 
Resolves #1127  
Resolves #1088 
Resolves #755 

Also fixes the issue with the CMS password.

### CODE REVIEW

File: app/views/catalog/_show_tools.html.erb [hunk 1]
```diff
@@ -40,13 +40,17 @@
       <li class="send_correction">
         <%= render "correction_form" %>
       </li>
-
+  <li class="librarian_view">
```
Comment (weiweishi) [inline app/views/catalog/_show_tools.html.erb]: There might be some conflict with what is in the master/tagged release, as the changes are done in https://github.com/ualbertalib/discovery/pull/1279 and https://github.com/ualbertalib/discovery/pull/1277, without the style. Did you pull the latest master for these changes?

File: config/initializers/comfortable_mexican_sofa.rb [hunk 2]
```diff
@@ -96,8 +96,8 @@
 
 # Default credentials for ComfortableMexicanSofa::AccessControl::AdminAuthentication
 # YOU REALLY WANT TO CHANGE THIS BEFORE PUTTING YOUR SITE LIVE
-ComfortableMexicanSofa::AccessControl::AdminAuthentication.username = 'ualibraries'
-ComfortableMexicanSofa::AccessControl::AdminAuthentication.password = 'discoverthi5'
+ComfortableMexicanSofa::AccessControl::AdminAuthentication.username = Rails.application.secrets.profiles_edit_user
```
Comment (weiweishi) [inline config/initializers/comfortable_mexican_sofa.rb]: Does this mean we will need to do the change in the config/secrets.yml template in the ansible playbook? Would be good to capture this in the release note/changelog?

File: app/views/catalog/_show_tools.html.erb (hunk 1 above)
Comment (nnunn) [reply app/views/catalog/_show_tools.html.erb]: right - I just noticed this. I didn't think I had committed that change. I'll pull and fix this.

File: config/initializers/comfortable_mexican_sofa.rb (hunk 2 above)
Comment (nnunn) [reply config/initializers/comfortable_mexican_sofa.rb]: No we dont' need to update secrets - it's already updated. This change should have been made in the code the last release, but was hard-coded by Neil instead.

Comment (weiweishi) [reply config/initializers/comfortable_mexican_sofa.rb]: 👍

**Comment by nnunn** [comment]
Yes it would. Do you want me to add that?

Also should I add the next release # to the CHANGELOG even if it isn't cut yet?

**Comment by weiweishi** [comment]
yes, I think Sean wants it to be pop-up window so if your code does that, that would be great!
And we'd usually add the changes to the "Unreleased" section for changes/addition (https://github.com/ualbertalib/jupiter/blob/master/CHANGELOG.md#unreleased), and will be a followup PR to change the "Unreleased" to the tagged version. We can still do that. I can open the PR for you to review/approve.  Or if you are okay with this, we can fast track this process, and add them all in at once.

**Comment by nnunn** [comment]
I've pushed that up.

I don't seem to have the CHANGELOG.md in my branch - I'm not sure why. I'm happy to update it but I'm not sure the best way :)

**Comment by weiweishi** [comment]
I will approve your PR and I will open a new one for you to approve with
changes to the Changelog!


*Weiwei Shi*Digital Initiative Applications Librarian

2-10L Cameron Library, University of Alberta
780-492-7802  |  weiwei.shi@ualberta.ca
*"The University of Alberta respectfully acknowledges that we are situated
on Treaty 6 territory, traditional lands of First Nations and Métis
people."*

**Pull request merged**